File: cmsong111/42576.py
#1 try
def solution(participant, completion):
    completion_len = len(completion)
    
    for i in range(completion_len):
        if completion[i] in participant:
            temp = participant.index(completion[i])
            participant.pop(temp)
    
    answer = participant[0]
        
    return answer



#2 try
def solution(participant, completion):
    participant.sort()
    completion.sort()
    
    answer = participant.pop()
    for i in range(len(completion)):
        if participant[i] != completion[i]:
            answer = participant[i] 
            break
   
    return answer

File: sudhdkso/42576.cpp
#include <string>
#include <vector>
#include <unordered_map>
using namespace std;

string solution(vector<string> participant, vector<string> completion) {
	string answer = "";
	unordered_map<string, int> hash;

	for (int i = 0;i < participant.size();i++) {
		if (hash.end() == hash.find(participant[i]))
			hash.insert({ participant[i], 1 });
		else
			hash[completion[i]]++;
	}


	for (int i = 0;i < completion.size();i++) {
		hash[completion[i]]--;
		if (hash[completion[i]] <= 0) {
			answer = completion[i];
			break;
		}

	}
	return answer;
}